Nos/Views/RepostButton.swift

```diff
@@ -71,6 +75,14 @@ struct RepostButton: View {
                 Task { await deleteReposts() }
             }
         }
+        .sheet(isPresented: $showQuotedNoteComposer, content: {
```

Comment:
I'm a fan of omitting the last parameter so that:

```
.sheet(isPresented: $showQuotedNoteComposer) {
    ....
}
```

and we have been using this style elsewhere.